src/temp/CarbonCapture/CarbonCapture.tsx

```diff
@@ -0,0 +1,192 @@
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import React from 'react';
+import style from './CarbonCapture.module.scss';
+import InfoIcon from '../icons/InfoIcon';
+import DownArrow from '../icons/DownArrow';
+import { useTranslation } from 'next-i18next';
+
+interface TabPanelProps {
+  index: number;
+  value: number;
+  beforeIntervation: string;
+  byProject: string;
+  sitePotential: string;
+}
+
+type carbonCapture = Omit<TabPanelProps, 'index' | 'value'>;
+
+function CustomTabPanel(props: TabPanelProps) {
+  const {
+    value,
+    index,
+    beforeIntervation,
+    byProject,
+    sitePotential,
+    ...other
+  } = props;
+  const { t } = useTranslation(['projectDetails']);
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      className={style.carbonCaptureMainContainer}
+      {...other}
+    >
+      {value === index && (
+        <>
+          <div className={style.carbonCaptureInfoContainer}>
+            <div className={style.carbonCaptureLabelMainContainer}>
+              <div className={style.carbonCaptureLabelContainer}>
+                <div className={style.carbonCapture}>
+                  {t('projectDetails:totalCO₂Captured')}
+                </div>
+                <div className={style.unit}>(tons)</div>
+              </div>
+              <div className={style.infoIconContainer}>
+                <InfoIcon />
+              </div>
+            </div>
+            <div className={style.carbonCatureDetailContainer}>
```

Comment:
Why have you provided static width here, won't this be dependent on the values received from api?